index.js

```diff
@@ -149,6 +149,20 @@ function init (ssb, config) {
     })
   )
 
+  pull(
+    listen.excludeMember(ssb),
+    pull.drain((msg) => {
+      const excludes = msg.value.content.excludes
+      const groupId = msg.value.content.recps[0]
```

Comment:
```suggestion
      const { excludes, recps: [groupId] } = msg.value.content
```
is it readable? I dunno...